test/Microsoft.TestPlatform.AcceptanceTests/RunsettingsTests.cs

```diff
@@ -220,18 +239,17 @@ public void RunSettingsWithParallelAndPlatformX64(RunnerInfo runnerInfo)
     public void RunSettingsWithInvalidValueShouldLogError(RunnerInfo runnerInfo)
     {
         SetTestEnvironment(_testEnvironment, runnerInfo);
-        using var tempDir = new TempDirectory();
 
         var runConfigurationDictionary = new Dictionary<string, string>
-                                                 {
-                                                         { "TargetPlatform", "123" }
-                                                 };
-        var runsettingsFilePath = GetRunsettingsFilePath(runConfigurationDictionary, tempDir);
+        {
+                { "TargetPlatform", "123" }
+        };
```

Comment:
Inner indent was "wrong"

```suggestion
        {
            { "TargetPlatform", "123" }
        };
```

Reply:
[optional] Maybe also use this syntax (clearer to me that's modifying a dictionary).

```suggestion
        {
            ["TargetPlatform"] = "123"
        };
```